src/03-modality-agnostic-files.md

```diff
@@ -14,20 +14,21 @@ Templates:
-| EthicsApprovals    | OPTIONAL              | [array][] of [strings][] | List of ethics committee approvals of the research protocols and/or protocol identifiers.                                                                                                                                                             |
-| ReferencesAndLinks | OPTIONAL              | [array][] of [strings][] | List of references to publications that contain information on the dataset. A reference may be textual or a [URI][uri].                                                                                                                               |
-| DatasetDOI         | OPTIONAL              | [string][]               | The Digital Object Identifier of the dataset (not the corresponding paper). DOIs SHOULD be expressed as a valid [URI][uri]; bare DOIs such as `10.0.2.3/dfjj.10` are [DEPRECATED][deprecated].                                                        |
+| **Key name**       | **Requirement level**              | **Data type**             | **Description**                                                                                                                                                                                                                                       |
+|--------------------|------------------------------------|---------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Name               | REQUIRED                           | [string][]                | Name of the dataset.                                                                                                                                                                                                                                  |
+| BIDSVersion        | REQUIRED                           | [string][]                | The version of the BIDS standard that was used.                                                                                                                                                                                                       |
+| HEDVersion         | RECOMMENDED                        | [string][]                | If HED tags are used: The version of the HED schema used to validate HED tags for study.                                                                                                                                                              |
+| DatasetLinks       | REQUIRED if [BIDS URIs][] are used | [object][] of [strings][] | Used to map a given `<dataset-name>` from a [BIDS URI][] of the form `bids:<dataset-name>:/absolute/path/within/dataset` to a local or remote location. The `<dataset-name>`: `local` is a reserved keyword that MUST NOT be a key in `DatasetLinks`  |
```

Comment:
This row is the only addition to the table, all other rows only show up as changed due to changes in the table pipes.

Reply:
Could you amplify a little bit about how the <dataset-name> is specified?  Overall, this is worthwhile.

Reply:
Note: I just had a call with @VisLab and the main take away was: this is good BUT the validator should *warn* if a user shares a dataset that points to a file *outside* of the shared dataset that is unavailable to third parties (e.g., because it is a file that is only on some local path of the machine where the data was curated)